toolbox/index.md

```diff
@@ -153,9 +153,9 @@ Project Management: GitHub
 ![GitHub](github.png){:width="264"}
 </figure>
 
-**What it is.** A project management tool tracks what needs to be done in the project, who’s working on what, and so forth. It also hosts the canonical [version](#version-control-systemvcs-git) of the code base.
+**What is it.** A project management tool tracks what needs to be done in the project, who’s working on what, and so forth. It also hosts the canonical [version](#version-control-systemvcs-git) of the code base.
 
-**Our Choice.** GitHub is the most popular Git host. It’s project-management tools are simple but cover everything you need for the group projects.
+**Our Choice.** GitHub is the most popular Git host. Its project-management tools are simple, but cover everything you need for the group projects.
```

Comment:
I’ll take the “its” fix (thanks, I hate those!). But I won’t take the comma, because I want the reader to read this in one breath.